src/state/hal.ts

```diff
@@ -259,7 +259,7 @@ function parseHalForms(context: string, body: hal.HalResource): ActionInfo[] {
 
   return Object.entries(body._templates).map( ([key, hf]) => {
     return {
-      uri: resolve(context, hf.target || ''),
+      uri: resolve(context, hf.target || body._links?.self?.href || ''),
```

Comment:
So it looks like this code will by default use the 'context' of the main document to resolve relative urls.

I think we either need to switch the context for these links to the self link for this to be consistent.

For example, if the embedded item has the URL `/foo/1`, and the target is `?bar`, then would expect `?bar` to be added to `/foo/1` or the parent doc?

This is not a rhetorical question, I wanna spend some time finding proof to figure out what the context for resolving relative paths should be here..

Reply:
Looks like https://rwcbook.github.io/hal-forms/#_code_target_code only expects absolute URIs for the `target` attribute.